gpii/configs/gpii.config.production.local.flowManager.revision.json5

```diff
@@ -0,0 +1,24 @@
+/**
+ * This configuration is for the production version of the system where the
+ * local flowManager provides its solutions registry dataSource with the url to
+ * the cloud based flowManager's /revision endpoint.
+ */
+{
+    "type": "gpii.config.production.local.flowManager.revision",
+    "options": {
+        "distributeOptions": {
+            "flowManager.defaultRevisionCloudURL": {
+                "record": "http://localhost:8084",
+                "target": "{that flowManager solutionsRegistryDataSource revisionRequester}.options.cloudURL"
```

Comment:
The `flowManager` in the target should be `gpii.flowManager.local` to be specific on distributing this option to LFM. Same for line 16.

Reply:
Right.

Reply:
Followup:   This distribution was originally part of "gpii.flowManager.config.untrusted.base.json5" in the flowmanager configs.  I used the record/target for the [settings data source](
https://github.com/GPII/universal/blob/master/gpii/node_modules/flowManager/configs/gpii.flowManager.config.untrusted.base.json5#L14) as the basis for revision requester's cloud url.  It also uses the expression `"{that flowManager ...}"` to pick out the LFM.  I have no objection to using "gpii.flowManager.local" here but is the other use okay?

Reply:
`gpii.flowManager.config.untrusted.base.json5` only deals with the local flow manager so there isn't a risk to accidentally distribute to both LFM and CBFM, but it's always better to be specific with the intended target.